index.md

```
@@ -0,0 +1,169 @@
+
+</style>
+
+<div id="header" class="cut1" markdown="1" style="background:#FFFFFF; color:#053C45;">
+
+<div id="header-inner" markdown="1">
+
+# Michael Noetel {#title}
+
+## Associate Professor, The University of Queensland {#subtitle}
+
+#### Psychology & AI Governance {#sub-subtitle}
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)